pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -70,6 +70,10 @@ func parseConfig(config *zconfig.EdgeDevConfig, getconfigCtx *getconfigContext,
 		// Network objects are used for systemAdapters
 		networksChanged := parseNetworkXObjectConfig(config, getconfigCtx)
 		if physioChanged || networksChanged {
+			// system adapter configuration that we send to zedrouter, depends
+			// on Physio configuration and Networks configuration. If either of
+			// Physio or Networks change, we should re-parse system adapters and
```

Comment:
Nit.. I would not specify zedrouter here.. We just say - we re-process sysAdapter config again.. We just publish it..  Zedagent doesn't know who the consumer is..

Reply:
will do